scipy/signal/tests/test_signaltools.py

```diff
@@ -2800,6 +2800,187 @@ def test_hilbert2_types(self, dtype):
         assert_equal(np.real(signal.hilbert2(in_typed)).dtype, dtype)
 
 
+class TestEnvelope:
+    """Unit tests for function ._signaltools.envelope()`. """
+    @classmethod
+    def raiseValErr(cls, match):
+        """Little helper to raise ValueError for mor concise code layout. """
+        return pytest.raises(ValueError, match=match)
```

Comment:
`mor` ->` `more` in the docstring above.

I wonder if we really want this shim though, the kwarg incantation of the function was obviously intended on the `pytest` side, and that's how we're using it elsewhere in SciPy. That said, pretty minor point!

Reply:
Thanks for taking a look, @tylerjereddy. Regarding the shim, I guess it is matter of personal taste. I don't really have strong a opinion on this. I removed the shim in commit [cbdc3ed](https://github.com/scipy/scipy/pull/20097/commits/cbdc3edd2c848118826dd6e3d082419da281877d).